AndroidProject/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,3 @@
 <resources>
-    <string name="app_name">Android Project</string>
+    <string name="app_name">AndroidProject</string>

```

Comment:
Please name app according to a repository name

Reply:
done